### ISSUE TITLE
Integrate MI/BI visualisation tools

### ISSUE THREAD

**Issue by Ed-Bajo** [ministryofjustice/analytical-platform#2954]
As a…
As an AP customer, I would like to publish visualisations based on AP data via the visualisation and dashboard tools. 

### Value / Purpose

The availability of a visualisation or dashboard tool will remove the need of building visualisation tools from the scratch.

It will make it easier, faster and better for people to create, share and consume trusted data, metrics and insights

### Useful Contacts

@julialawrence
@Ed-Bajo 

### User Types

AP Customers

### Proposal

During our discovery and recent reprioritisation exercises, it was identified that one of present needs of the Analytical Platform users are dashboard and visualisation tools. In addition, it was also established that there was a need to expand our tooling offerings on the platform (e.g. PowerBI, Quicksight, etc).  

The team's focus is to carry out discovery to ascertain users needs and business requirements; in respect of visualisation and dashboard tools. In parallel, the team will continue to explore options to expand our tooling offerings for the AP by conducting technical spikes and proof of concept to establish the technical feasibility of integrating new analytical tools onto the platform (e.g. Quicksight, etc).

Components to create are as follows:

### Additional Information

The workflow mentions a critical assumption that all readers/authors of the report have the same set of permissions. This assumption isn't correct, however, the requestor will appoint a master author of the report who will have access to the full data and then will utilise capability native to PowerBI to partition the report with smaller subsets of data. As members of the DPAT team, we are not implementing that portion of the solution: that will sit with the customer.

Out of scope: High Availability and Scalability are out of scope.

### Definition of Done

- [ ] 
- [ ] 
- [ ] 
- [ ]
